wandb/apis/public/utils.py

```diff
@@ -66,3 +68,38 @@ def parse_org_from_registry_path(path: str, path_type: PathType) -> str:
         if is_artifact_registry_project(project):
             return org
     return ""
+
+
+def fetch_org_from_settings_or_entity(
+    settings: dict,
+    default_entity: str | None,
```

Comment:
```suggestion
    default_entity: Optional[str] = None,
```
Without `Optional`, i was running into build issues

Reply:
ah yeah i wasn't considering python versions. thanks